pages/1.11/installing/oss/troubleshooting/index.md

```diff
@@ -54,7 +54,7 @@ oss: true
 
 1. Ensure that firewalls and any other connection-filtering mechanisms are not interfering with cluster component communications. TCP, UDP, and ICMP must be permitted.
 
-   Ensure that services that bind to port `53`, which is required by DNS Forwarder (`dcos-spartan.service`), are disabled and stopped. For example:
+   Ensure that services that bind to port `53`, which is required by DNS Forwarder (`dcos-dcos-net.service`), are disabled and stopped. For example:
```

Comment:
minor typo ... it should be dcos-net and not dcos-dcos-net

Reply:
Fixed!